packages/SketchMorph2-Core.package/MorphicMonet.class/instance/chooseColor.st

```diff
@@ -1,7 +1,9 @@
 tooling
 chooseColor
-	| currentColor |
-	self world activeHand showTemporaryCursor: (CursorWithAlpha fromIcon: UiFugueIcons pipetteIcon).
+	| currentColor pipetteTool |
+	pipetteTool := self menuBrushCollection last.
+	self activeHand showTemporaryCursor:
+		(CursorWithAlpha fromIcon: pipetteTool icon).
```

Comment:
I don't see the added value in doing this. In fact, by doing so you assume the pipette tool will always be the last element in the brush menu which might not always be the case.